### ISSUE TITLE
[Linux]: Game doesn't launch using primusrun %command% after last system update

### ISSUE THREAD

**Issue by erfoon** [ValveSoftware/Dota-2#1737]
#### Your system information

* System information from steam:
OS Linux Manjaro
 [System Info](https://gist.github.com/erfoon/a47b7789a0c66444f9917d90de29ed85)
* Have you checked for system updates?: [Yes]
* Are you using the latest stable video driver available for your system? [Yes]
* Have you verified the game files?: [Yes]

#### Please describe your issue in as much detail as possible:
I always run Dota 2 using bumblebee by putting `primusrun %command%` in lunch options of the game. However, after last system update Dota doesn't launch anymore. If I remove the primusrun it does but with FPS 10 which is almost unplayable. 

I tried to open the whole steam with primusrun but I get the famous fatal error: `failed to load steamui.so manjaro`

#### Steps for reproducing this issue:

1. Update Manjaro
2. Use Primus for launching Dota
3. Game won't open

**Comment by kisak-valve** [comment]
Hello @erfoon, what video card / driver version are you using? If you're using an nVidia card that is supported by the 435+ driver series, then nVidia's [prime render offloading](https://download.nvidia.com/XFree86/Linux-x86_64/435.17/README/primerenderoffload.html) is an option you could try with `__NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia %command%` in the game's launch options.

**Comment by erfoon** [comment]
@kisak-valve thanks for response, I have a `GM108M [GeForce 840M]` graphic card with `intel-nvidia-430xx-bumblebee` driver. I set your suggestion in launch option and the result was this error: `Failed to create an OpenGL context. Your graphics card must support at least OpenGL v3.1.`

**Comment by kisak-valve** [comment]
Right, and the 430.xx driver series predates the newer method of render offloading.

**Comment by erfoon** [comment]
@kisak-valve So you don't have any workaround for that?
